src/lib/authserver/ui/new_dir_inst.rb

```diff
@@ -120,7 +124,7 @@ def ok_handler
     # Always remove the ini file
     DS389.remove_setup_ini
     if !ok
-      Popup.Error(_('Failed to set up new instance! Log output may be found in /var/log/YaST/y2log'))
+      Popup.Error(_('Failed to set up new instance! Log output may be found in %{y2log}'), y2log: y2log)
```

Comment:
Missing [`format`](https://ruby-doc.org/core-3.1.2/Kernel.html#method-i-format)

```suggestion
      Popup.Error(format(_('Failed to set up new instance! Log output may be found in %{y2log}'), y2log: y2log))
```